<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="glob">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>服务器PORT</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="port"/>
          </item>
          <item>
           <widget class="QLineEdit" name="zhuangtai">
            <property name="maximumSize">
             <size>
              <width>65</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>目标IP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ip"/>
          </item>
          <item>
           <widget class="QPushButton" name="setip">
            <property name="minimumSize">
             <size>
              <width>65</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>65</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>确认</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>目标PORT</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="port1"/>
          </item>
          <item>
           <widget class="QPushButton" name="setport">
            <property name="minimumSize">
             <size>
              <width>65</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>65</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>确认</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="lobb">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="1" column="0">
              <widget class="QTextEdit" name="textEdit"/>
             </item>
             <item row="0" column="0">
              <widget class="QPushButton" name="jieshou">
               <property name="text">
                <string>接收</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="2" column="0">
              <widget class="QTextEdit" name="textEdit_2"/>
             </item>
             <item row="0" column="0">
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>发送</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
